game/__main__.py

```diff
@@ -54,6 +54,14 @@ def setup(self):
         self.set_viewport_on_player()
         self._draw()
 
+    def win_level(self):
+        # TODO: Transition to next level
+        print("You are built different")
+
+    def lose_level(self):
+        # TODO: Lose Ui
```

Comment:
i think we agreed to not have a loseview, it's easier to just reset the level

Reply:
Fair, we can just leave it as is